docs/src/PolyhedralGeometry/pg_linear_programs.md

```diff
@@ -10,11 +10,109 @@ using Oscar
 Pages = ["pg_linear_programs.md"]
 ```
 
-# Linear programs
+# Linear Programs
+
 
 ## Introduction
 
+The task of a linear program is to optimize a linear function over a
+polyhedron.
+
+
+
 ## Constructions
 
+Linear programs are constructed by passing a polyhedron, and a linear function
+in terms of a vector and translation. One may specify whether the optimization
+problem is in terms of maximum or minimum.
```

Comment:
For my feeling it is okay here to leave the default out because the docstring follows right after and should contain this information in its header.

I still agree with the improved phrasing.